data/pt_PT/system/common-ui/lib/common-ui-5.x_jar/org/pentaho/common/ui/messages/messages_pt_PT.properties

```diff
@@ -0,0 +1,72 @@
+#
```

Comment:
The "jar folders"  *-5.x_jar lost the version digit:

common-ui-5.x_jar -> common-ui_jar

You probably made your changes on top on an older master.